### PR TITLE
Update db-schema.md

### PR DESCRIPTION
**Pull request by itfork**
## Purpose of this pull request

This pull request (PR) corrects path to XSD file in case of Composer.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.html

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@itfork thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md [hunk 1]
```diff
@@ -154,8 +154,8 @@ A column can have the following attributes:
 
 For more information about each type, refer to the annotations in the corresponding XSD file. The location of the XSD file depends on how you installed Magento.
 
-*  [Archive download]({{page.baseurl}}/install-gde/prereq/zip_install.html): `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc`
-*  [Composer]({{page.baseurl}}/install-gde/composer.html) or [GitHub]({{page.baseurl}}/install-gde/prereq/dev_install.html) installation: `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`
+*  [Archive download]({{page.baseurl}}/install-gde/prereq/zip_install.html) or [Composer]({{page.baseurl}}/install-gde/composer.html): `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc`
```
Comment (rogyar) [inline src/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]: Hi @itfork. Thank you for your contribution. 

With Composer installation of the OpenSource edition, you have the XSD files in  `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`. So the current statement is correct.

To double-check this fact you may run the following command in some empty directory:

`composer create-project magento/community-edition`

Comment (itfork) [reply src/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md:157]: I'm sorry, I didn't know

**Pull request closed without merging**